e2e-tests/cypress/tests/cypress/e2e/models/LoginPage.ts

```diff
@@ -4,6 +4,7 @@ export class LoginPage {
   // selectors
   emailInput = "input[type=email]";
   passwordInput = "input[type=password]";
+  forgotPasswordLink = '[data-test="forgot-password-link"]';
   submitBtn = "[type=submit]";
   emailHint = "#vee_Email";
   passwordHint = "#vee_Password";
```

Comment:
Would prefer ' instead of " like you did line 7
```suggestion
  emailInput = 'input[type=email]'
  passwordInput = 'input[type=password]'
  forgotPasswordLink = '[data-test="forgot-password-link"]'
  submitBtn = '[type=submit]'
  emailHint = '#vee_Email'
  passwordHint = '#vee_Password'
```